program/freedom-metal-program.wake

```diff
@@ -1,4 +1,28 @@
 # FIXME
+def addMetalMemory type memSubnode inDTSContents =
+  def subnodeOpt =
+    match "({memSubnode}@[0-9]+)".stringToRegExp
```

Comment:
I'm assuming that this is supposed to match one of the DTS nodes names that look like `clint@2000000`. If so, then the address on the right side is a hex number and you need to include `a-z` in that pattern.